src/ctia/task/migration/migrate_es_stores.clj

```diff
@@ -269,25 +278,30 @@
     (System/exit -1)
     (System/exit 0)))
 
-(defn run-migration
-  [migration-id prefix migrations store-keys batch-size buffer-size confirm? restart?]
-  (assert migration-id "Please provide an unique ID for this migration process")
+(s/defn ^:always-validate check-migration-params
+  [{:keys [prefix
+           store-keys]} :- MigrationParams]
+  (doseq [store-key store-keys]
+    (let [index (get-in @properties [:ctia :store :es store-key :indexname])]
+      (assert (not= (mst/prefixed-index index prefix)
```

Comment:
throwing AssertionError by hand may be better here since assert depends on `*assert*` which ideally would be set to false in production (I think currently it's not).

i.e. given you `^:always-validate`, it seems well-aligned to not rely on `*assert*` being bound to its default val